composer.json

```diff
@@ -48,6 +48,16 @@
         "a9f/fractor-yaml": "self.version",
         "a9f/typo3-fractor": "self.version"
     },
+    "replace": {
```

Comment:
@simonschaufi why did you add exactly the same "replace" key again?

Reply:
that was not intentional but happened because of the rebase. I'll fix this.